tests/integration/components/questionnaire/dds-project-field-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import StoreStub from "../../../helpers/store-stub";
+
+moduleForComponent('questionnaire/dds-project-field', 'Integration | Component | questionnaire/dds project field', {
+  integration: true,
+  beforeEach: function() {
+    this.register('service:store', StoreStub);
+  }
+});
+
+test('it renders', function(assert) {
```

Comment:
Looks like this test case is just the auto-generated one for the new component, but it's not testing any of the code in that component. Are there any behaviors tested in `file-group-list-test.js` that should now be here in `dds-project-field-test.js`?

I think an integration test can still work here, since rendering should invoke `didInsertElement()` and trigger the services. But in some cases if I just want to test the JS code of a component, I'll do that in a unit test. Ember doesn't generate unit tests for components automatically, but there's a `component-test` generator that can conjure one up:

```
$ ember g component-test dds-project-field -u
installing component-test
  create tests/unit/components/dds-project-field-test.js
```

Reply:
This component doesn't actually display anything so I added a unit test for didInsertElement.